.storybook/docs-root-v9.css

```diff
@@ -0,0 +1,23 @@
+/* remove the docs wrapper bg to let page bg show through */
```

Comment:
This component was copied over from https://github.com/microsoft/fluentui/blob/master/apps/public-docsite-v9/.storybook/docs-root-v9.css and will be removed once original file will be published as a part of a new package.